docs/smart-contracts/creating.md

```diff
@@ -0,0 +1,53 @@
+---
+title: Creating smart contracts
+author: 'Yuxin Li'
+lastUpdated: 6th November 2023
+---
+## Introduction
+Creating smart contracts on blockchain making sure faster execution and reduced costs by eliminating the need for intermediaries. This guide provides step-by-step instructions for creating smart contracts on the Tezos blockchain. You can find additional resources on testing and deploying your smart contract in subsequent chapters.
+
+## Choosing your smart contract language
+Tezos supports a variety of smart contract languages:
```

Comment:
You can link to the languages page directly and just talk about how to select a language and if different languages have different capabilities or features here.